src/common/flowConfig.ml

```diff
@@ -61,6 +62,7 @@ let default_module_system = Node
 let default_options root = {
   moduleSystem = default_module_system;
   module_name_mappers = [];
+  module_file_exts = [];
```

Comment:
`module_file_exts` is an empty list by default, it allows the user to choose if `.js` or `.jsx` will be checked or not, if not `module.file_ext` are specified in `[options]` then `module_file_exts = [".js"; ".jsx"]`.
But if you want to force the user to has its `.js` and `.jsx` to be checked, I can do it.
What are you thoughs ?